client/src/api/category.ts

```diff
@@ -0,0 +1,31 @@
+import axios from './axios';
+
+export default class Category {
```

Comment:
please no class in the front end, check the auth api file and follow the same code style